components/automate-cli/pkg/diagnostics/integration/iam_projects.go

```diff
@@ -162,16 +182,32 @@ func CreateIAMProjectsDiagnostic() diagnostics.Diagnostic {
 				return err
 			}
 
+			loaded := iamProjectSave{}
+			err = tstCtx.GetValue("iam-projects", &loaded)
+			if err != nil {
+				return errors.Errorf(err.Error(), "could not find generated context")
+			}
+
 			tstCtx.SetValue("iam-projects", &iamProjectSave{
-				ID:     projectInfo.Project.ID,
-				RuleID: projectInfo.Project.Rule.ID,
+				ID:      projectInfo.Project.ID,
+				RuleID:  projectInfo.Project.Rule.ID,
+				Skipped: loaded.Skipped,
 			})
 			return nil
 		},
 		Verify: func(tstCtx diagnostics.VerificationTestContext) {
 			loaded := iamProjectSave{}
 			err := tstCtx.GetValue("iam-projects", &loaded)
 			require.NoError(tstCtx, err, "Could not find generated context")
+			if loaded.Skipped == true {
+				// this happens in the v1->v2 force upgrade scenario:
+				// when we run generate diagnostic data while on v1
+				// then force-upgrade to v2
+				// then run verify and cleanup on that data
+				// for projects, since the Generate step was skipped
+				// there is nothing to verify on v2
```

Comment:
Took me awhile to parse this; see if I got it right:
```suggestion
				// this happens in the v1->v2 force upgrade scenario:
				// when we run generate diagnostic data while on v1
				// then force-upgrade to v2
				// then run verify and cleanup on that data.
				// Since the Generate step was skipped for projects
				// there is nothing to verify on v2.
```